android/src/main/java/com/wultra/android/powerauth/reactnative/PowerAuthRNModule.java

```diff
@@ -364,26 +364,28 @@ public void commitActivation(String instanceId, final ReadableMap authMap, final
         this.usePowerAuthOnMainThread(instanceId, promise, new PowerAuthBlock() {
             @Override
             public void run(@NonNull PowerAuthSDK sdk) {
-                PowerAuthAuthentication auth = constructAuthentication(authMap);
-                if (auth.usePassword == null) {
-                    promise.reject(EC_PASSWORD_NOT_SET, "Password is not set.");
+                final PowerAuthAuthentication auth = constructAuthentication(authMap, true, promise);
+                if (auth == null) {
                     return;
                 }
                 if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.M && authMap.getBoolean("useBiometry")) {
                     String title = authMap.getString("biometryTitle");
                     if (title == null) {
-                        title = " "; // to prevent crash
+                        title = "null"; // to prevent crash
```

Comment:
Is to goal to force an integrator to use title and message? Otherwise, this will be displayed in the Android UI dialog.

Reply:
Title & message is mandatory on Android. Developer should always provide some sane values, except the asymmetric key is used behind the scene (it doesn't require user's authentication on key setup)